docs/developer/hacking.rst

```diff
@@ -0,0 +1,51 @@
+#######
+Hacking
+#######
+
+.. note:: Before reading this file, please read the :doc:`kytos:contributing`
+    section that contains the main guidelines of the project.
+
+Development Environment Setup
+*****************************
+
+It is possible to install any of the *kytos* repos in develop mode, so that the
+installed files are kept in the original location, and only links are made to
+the system locations. This allows developpers to quickly test changes made in
+source code.
+
+To install a repo in develop mode, inside the repo source folder run:
+
+	.. code:: shell
+
+	    sudo python setup.py develop
```

Comment:
@cemsbr, I reviewed it on quickstart for ex.
Still see no reason not to call setup. Result is the same, pip use setuptools.
On this same page we call setup to do the tests for ex.
I made the changes anyway.